config/config.go

```diff
@@ -279,6 +273,16 @@ func (c *Config) dataValidIntervalConfigLoad() int64 {
 	return c.DataValidInterval
 }
 
+func (c *Config) remappingGasPriceOracleWithDomainIDs() map[int][]Oracle {
```

Comment:
since fee oracle is using yaml as config file, the domainID list loaded from file is not a list naturally,  so this func remapping the static domain list in yaml to a map, if you have a better way I can change it, but afaik this is limited by yaml

Reply:
wait a sec, I prob know how to do it, will try